### ISSUE TITLE
Regexp capture does not accept an array

### ISSUE THREAD

**Issue by jamesfiltness** [shoreditch-ops/artillery#96]
It is only possible to pass a single object to capture. It would be nice if you could pass an array of objects to capture and have them all run, i.e.:

```
"capture":[{
  "regexp": "<input name=\"some-name\" type=\"text\" value=\"(.*?)\" ",
  "as": "someValue"
},{
  "regexp": "<input name=\"some-other-name\" type=\"text\" value=\"(.*?)\" ",
  "as": "someOtherValue"
}]
```

**Comment by hassy** [comment]
Good idea 👍

**Comment by hassy** [comment]
Fixed in `v1.5.0-12`

https://github.com/shoreditch-ops/artillery/releases/tag/1.5.0-12
